mne/decoding/tests/test_time_gen.py

```diff
@@ -424,4 +424,22 @@ def test_decoding_time():
     tg = TimeDecoding(clf=clf, cv=cv)
     tg.fit(epochs, y=y)
 
+
+@requires_sklearn
+def test_string_scorer():
+    """Test whether passing string as scorer works as intended"""
```

Comment:
"Test whether scorer can be defined with a string that refers to a scikit-learn scorer"